components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2799,18 +2644,16 @@ end subroutine get_cloud_dsd2
 
 
   !===========================================================================================
-  subroutine get_rain_dsd2(qr,nr,mu_r,rdumii,dumii,lamr,mu_r_table,cdistr,logn0r,rcldm)
+  subroutine get_rain_dsd2(qr,nr,mu_r,lamr,cdistr,logn0r,rcldm)
```

Comment:
Lines: 2675-2689, i think we can go ahead and delete these as long as we cleaning up the code too.

```
       ! AaronDonahue: Comment out the variable mu_r calculation below.
!       if (inv_dum.lt.282.e-6) then
!          mu_r = 8.282
!       elseif (inv_dum.ge.282.e-6 .and. inv_dum.lt.502.e-6) then
!          ! interpolate
!          rdumii = (inv_dum-250.e-6)*1.e+6*0.5
!          rdumii = max(rdumii,1.)
!          rdumii = min(rdumii,150.)
!          dumii  = int(rdumii)
!          dumii  = min(149,dumii)
!          mu_r   = mu_r_table(dumii)+(mu_r_table(dumii+1)-mu_r_table(dumii))*(rdumii-  &
!               real(dumii))
!       elseif (inv_dum.ge.502.e-6) then
!          mu_r = 0.
!       endif
```